bin/htmlhint

```diff
@@ -300,12 +300,13 @@ function getConfig(configPath, base, formatter){
         if(fs.statSync(base).isDirectory() === false){
             base = path.dirname(base);
         }
-        while(base){
-            var tmpConfigFile = path.resolve(base+path.sep, '.htmlhintrc');
+        while(true){
```

Comment:
Yes, maybe that works
But can you rewrite this in a way that does not use an always `true` statement in the while clause?

This conflicts with the eslint rule [no-constant-condition](https://eslint.org/docs/rules/no-constant-condition) and we want that later in our project